pop-api/examples/fungibles/tests.rs

```diff
@@ -0,0 +1,831 @@
+use super::*;
+use drink::{
+	sandbox_api::assets_api::AssetsAPI,
+	session::{ContractBundle, Session},
+};
+use frame_support::assert_ok;
+use pop_api::{
+	primitives::{
+		ArithmeticError::Overflow,
+		Error::{Arithmetic, Module},
+		TokenId,
+	},
+	v0::fungibles::events::{Approval, Created, Transfer},
+};
+use pop_sandbox::{Balance, DevnetSandbox as Sandbox, ALICE, BOB};
+use scale::Encode;
+use std::sync::LazyLock;
+use utils::*;
+
+#[drink::contract_bundle_provider]
+enum BundleProvider {}
+
+const AMOUNT: Balance = MIN_BALANCE * 4;
+const MIN_BALANCE: Balance = 10_000;
+const TOKEN: TokenId = 1;
+
+static CONTRACT: LazyLock<ContractBundle> = LazyLock::new(|| BundleProvider::local().unwrap());
+
+/// Deployment and constructor method tests.
+
+#[drink::test(sandbox = Sandbox)]
```

Comment:
```suggestion
#[drink::test(sandbox = Pop)]
```
Would read nicer imo.